docs/modules/integrations/pages/github-actions/upload-an-app.adoc

```diff
@@ -0,0 +1,15 @@
+= Upload an app using GitHub Actions
+:navtitle: Upload an app
+
+Learn how to upload an app to your app repository using GitHub Actions.
+
+== Before you start
+
+You'll need the following:
+
+* Kobiton username and API key. To obtain them, follow the instructions in xref:profile:manage-your-api-credentials.adoc#_get_an_api_key[this guide].
+
+== Upload the app
+
+To upload an app to Kobiton using GitHub Actions, follow the instruction on our link:https://github.com/marketplace/actions/kobiton-upload-mobile-app#kobiton-action-for-upload-mobile-app[official GitHub Marketplace Actions] page.

```

Comment:
Suggestion to reduce words and focus on what it's called there: 

Follow the instructions posted on GitHub Marketplace for the link:https://github.com/marketplace/actions/kobiton-upload-mobile-app#kobiton-action-for-upload-mobile-app[Kobiton Action for Upload Mobile App].

Reply:
Also - what is our policy or what should our policy be for opening in a new window - should this open in a new window? (And what happens if we do that on the widget?)

Reply:
I'll update the policy somewhere else, but let me try opening this on a new window and see what happens on the widget.

Reply:
@Hikkigaya please work on the wording change and open the link in the new window.

Reply:
Okay, will do

Reply:
Updated